Create zh_cn.json

QiuShui1012 opened the pull request.

QiuShui1012 committed:
Create zh_cn.json

ChefMooon commented:
I forgot to mention I added Stair and Slab variants. Would you be able to add those when you are available? I will do another small release once they are ready. Appreciate the translations!

QiuShui1012 committed:
Merge branch 'ChefMooon:1.21.x-multi' into patch-1

QiuShui1012 committed:
Update zh_cn.json for slabs and stairs

QiuShui1012 commented:
> I forgot to mention I added Stair and Slab variants. Would you be able to add those when you are available? I will do another small release once they are ready. Appreciate the translations!

I have updated the translation.

ChefMooon commented:
Thank you!
